libs/database/src/collab/collab_db_ops.rs

```diff
@@ -76,11 +76,11 @@ pub async fn insert_into_af_collab(
       if existing_workspace_id == workspace_id {
         sqlx::query!(
           "UPDATE af_collab \
-        SET blob = $2, len = $3, partition_key = $4, encrypt = $5, owner_uid = $6 WHERE oid = $1",
```

Comment:
We cannot update `partition_key` as it's a part of the primary key on `af_collab` table. This would cause all foreign keys to crash.